# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1522105406 25200
#      Mon Mar 26 16:03:26 2018 -0700
# Branch stable
# Node ID b01409e9dfcc226fc7d2fdc18ce6c1eb7cc9fad0
# Parent  92252c8c755966479288e36c9a2b15a3c5a88268
doc: include macros.texi when generating standalone BUGS file

* bugs.txi: Include macros.texi when generating standalone BUGS file.

diff --git a/doc/interpreter/bugs.txi b/doc/interpreter/bugs.txi
--- a/doc/interpreter/bugs.txi
+++ b/doc/interpreter/bugs.txi
@@ -20,16 +20,17 @@
 @c distribution, as well as in the Octave manual.
 
 @ifclear BUGSONLY
 @node Trouble
 @appendix Known Causes of Trouble
 @end ifclear
 
 @ifset BUGSONLY
+@include macros.texi
 
 This file documents known bugs in Octave and describes where and how to
 report any bugs that you may find.
 
 Copyright (C) 1996, 1997, 2007 John W. Eaton.  You may copy, distribute, and
 modify it freely as long as you preserve this copyright notice and
 permission notice.
 
